### DISCUSSION TITLE
Any way to generate 'missing' lists from a global view, rather than library-level?

### DISCUSSION THREAD

**Discussion by EnSt0rm** [meisnate12/Plex-Meta-Manager#2350] [Q&A]
For a few years, I've had Plex set up with separate libraries for Kids content vs non-Kids content.

Since using Kometa, I find that some of my collections contain both kids and non-kids content.
For example, the Star Wars collection has most movies and TV shows that are not part of the kids collection, but then a few specific shows are definitely more for Kids and have been placed in the Kids libraries, e.g. 'Young Jedi Adventures' and 'Star Wars Resistance'.

When I run Kometa, I want to know when items are missing from the Star Wars collection overall, e.g. when a brand new show is released, and not found across **all** of my libraries, but I don't want to know where a show is missing from Library A (non-kids TV), but found in Library B (Kids TV).

Is there any way for Kometa to look at all libraries (global vs per-library) before treating an item as missing?

**Comment by chazlarson** [comment]
There is not presently a way to do this.
